pkg/util/address_family_test.go

```diff
@@ -0,0 +1,58 @@
+package util
+
+import (
+	"errors"
+	kubeovnv1 "github.com/kubeovn/kube-ovn/pkg/apis/kubeovn/v1"
+	"golang.org/x/sys/unix"
+	"testing"
+)
+
+func TestProtocolToFamily(t *testing.T) {
+	tests := []struct {
+		name string
+		prot string
+		want int
+		err  error
+	}{
+		{
+			name: "correct",
+			prot: kubeovnv1.ProtocolIPv4,
+			want: unix.AF_INET,
+			err:  nil,
+		},
+		{
+			name: "v6",
+			prot: kubeovnv1.ProtocolIPv6,
+			want: unix.AF_INET6,
+			err:  nil,
+		},
+		{
+			name: "dual",
+			prot: kubeovnv1.ProtocolDual,
+			want: unix.AF_UNSPEC,
+			err:  nil,
+		},
+		{
+			name: "err",
+			prot: "damn",
+			want: unix.AF_UNSPEC,
+			err:  errors.New(""),
+		},
+	}
+	for _, c := range tests {
+		t.Run(c.name, func(t *testing.T) {
+			ans, err := ProtocolToFamily(c.prot)
+			if c.err == nil && err != nil {
+				t.Fatalf("%v expected %v, but %v, %v got",
```

Comment:
why use `t.Fatalf` here and `t.Errorf` in other places?

Reply:
fixed...copied from previous work and didn't notice that diff.